sanityv3/schemas/documents/header/sharedHeaderFields.ts

```diff
@@ -199,8 +199,10 @@ const heroImage = {
   validation: (Rule: Rule) =>
     Rule.custom((value: ImageWithAltAndCaption, context: ValidationContext) => {
       const { parent } = context as unknown as DocumentType
+      //@ts-ignore:add _type?
       if (
-        (parent?.heroType === HeroTypes.FIFTY_FIFTY || parent?.heroType === HeroTypes.FULL_WIDTH_IMAGE) &&
+        (parent?.heroType === HeroTypes.FIFTY_FIFTY ||
+          (parent?._type !== 'homePage' && parent?.heroType === HeroTypes.FULL_WIDTH_IMAGE)) &&
```

Comment:
Need not check explicitly here... Editors can choose default hero and not choose an image.